src/HideContentInPostPreviews.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of jslirola/flarum-ext-login2seeplus.
+ *
+ * Copyright (c) 2020
+ * Original Extension by WiseClock
+ * Updated by jslirola
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JSLirola\Login2SeePlus;
+
+class HideContentInPostPreviews extends FormatContent
+{
+    public function getContent($serializer, $attributes)
```

Comment:
Rename this to `__invoke`,  and add $model as a middle argument, and return attributes at the end (or split that into 2 methods), then in the extender, you can just do: 
`->mutate(HideContentInPostPreviews)`